bin/test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# Print job information.
+
+# Initialize Gradle and print version.
+./gradlew --version
+
+# Run all checks.
+# Note: This will also initialize the Gradle daemon, which is important,
+# because, otherwise, the next `printVersion` command will splatter the
+# stdout with corresponding messages.
+./gradlew check
```

Comment:
I did not have any luck with `--no-daemon`, because it would produce the exact problem as outlined at https://github.com/crate/cratedb-flink-jobs/pull/6/files#r1146681656 _on my machine instead_. Instead of honoring `--no-daemon`, it does exactly the opposite, and spins up a daemon! I am baffled.